tests/ctst/world/Zenko.ts

```diff
@@ -546,15 +651,18 @@ ${JSON.stringify(policy)}\n${err.message}\n`);
     async prepareIamUser() {
         this.saved.userName = `iamusertest${Utils.randomString()}`;
         // Create IAM user
-        this.addCommandParameter({ userName: this.saved.userName });
+        this.addCommandParameter({ userName: this.saved.userName as string });
         await IAM.createUser(this.getCommandParameters());
         this.resetCommand();
         // Create credentials for the user
-        this.addCommandParameter({ userName: this.saved.userName });
+        this.addCommandParameter({ userName: this.saved.userName as string });
         const accessKey = await IAM.createAccessKey(this.getCommandParameters());
-        this.parameters.IAMSession = JSON.parse(accessKey.stdout).AccessKey;
-        this.cliMode.parameters.IAMSession = this.parameters.IAMSession;
-        CacheHelper.parameters.IAMSession = this.parameters.IAMSession;
+        this.parameters.IAMSession =
+            (JSON.parse(accessKey.stdout) as { AccessKey: string })?.AccessKey;
+        this.cliMode.parameters.IAMSession =
+            this.parameters.IAMSession as ClientOptions['IAMSession'];
+        (CacheHelper.parameters as { IAMSession: ClientOptions['IAMSession'] }).IAMSession =
+            this.parameters.IAMSession as ClientOptions['IAMSession'];
```

Comment:
`ClientOptions['IAMSession']` here to me is an object as follows:
```
IAMSession?: {
        AccessKeyId: string;
        SecretAccessKey: string;
        SessionToken: string;
    };
```
Am I missing something?

Reply:
You can define this type in `ZenkoWorldParameters` to avoid multiple `as`, in general it's better to define the variables with their type than casting them everytime we use them

Reply:
b7460b1c75e76aefa5df57099a8cbecb7c46e999